include/mbedtls/ctr_drbg.h

```diff
@@ -281,8 +285,8 @@ int mbedtls_ctr_drbg_random( void *p_rng,
  * \param ctx           The CTR_DRBG context.
  * \param path          The name of the file.
  *
- * \return              \c 0 on success,
- *                      #MBEDTLS_ERR_CTR_DRBG_FILE_IO_ERROR on file error, or
+ * \return              \c 0 on success.
+ * \return              #MBEDTLS_ERR_CTR_DRBG_FILE_IO_ERROR on file error, or
```

Comment:
We should be uniform as to whether all failure codes are described in a single `\return` block or not. E.g. [here](https://github.com/ARMmbed/mbedtls/pull/1512/files#diff-e2cc6b67dc588f8430431295c5f54f87R194) there are separate `\return` statements for different kinds of error codes.

Reply:
I agree, as we discussed over the phone. The issue is that you sometimes define the source of the error, and sometimes you do not. Your team has also objected to the use of "on failure for any other reason", so that is is difficult to separate statements where the reason for failure is not clearly detailed.

Reply:
@RosZad Yes, I think this needs to be discussed with the entire team.